src/utils/call_responses.ts

```diff
@@ -0,0 +1,47 @@
+import {AppCallResponse, AppForm} from 'mattermost-redux/types/apps';
+import {AppCallResponseTypes} from 'mattermost-redux/constants/apps';
+
+export type FieldValidationErrors = {[name: string]: string};
+
+export function newOKCallResponse(): AppCallResponse {
+    return {
+        type: AppCallResponseTypes.OK,
+    };
+}
+
+export function newOKCallResponseWithMarkdown(markdown: string): AppCallResponse {
+    return {
+        type: AppCallResponseTypes.OK,
+        markdown,
+    };
+}
+
+export function newOKCallResponseWithData(data: {}): AppCallResponse {
+    return {
+        type: AppCallResponseTypes.OK,
+        data,
+    };
+}
+
+export function newFormCallResponse(form: AppForm): AppCallResponse {
+    return {
+        type: AppCallResponseTypes.FORM,
+        form,
+    };
+}
+
+export function newErrorCallResponseWithMessage(error: string): AppCallResponse {
```

Comment:
nit: rename `error` to `message` (to distinguish from the following `newErrorCallResponseWithFieldErrors`)

Reply:
fair enough. I'll explicitly use the `error` field with value `message`